common/src/main/java/com/wynntils/screens/settings/lists/FeatureList.java

```diff
@@ -0,0 +1,262 @@
+    }
+
+    @Override
+    public int getRowWidth() {
+        return settingsScreen.width / 6;
+    }
+
+    @Override
+    public int getRowLeft() {
+        return this.x0 + settingsScreen.width / 90;
+    }
+
+    @Override
+    public boolean mouseClicked(double mouseX, double mouseY, int button) {
+        float scrollButtonXPos = getScrollButtonXPos();
+        float scrollButtonYPos = getScrollButtonYPos();
+
+        int size = (int) (settingsScreen.width / 65f);
+
+        // Check if we clicked on the scroll button
+        if (mouseX >= scrollButtonXPos
+                && mouseX <= scrollButtonXPos + size
+                && mouseY >= scrollButtonYPos
+                && mouseY <= scrollButtonYPos + size) {
+            draggingScrollButton = true;
+            return false;
+        }
+
+        return super.mouseClicked(mouseX, mouseY, button);
+    }
+
+    @Override
+    public boolean mouseDragged(double mouseX, double mouseY, int button, double dragX, double dragY) {
+        if (draggingScrollButton) {
+            float height = getRenderHeight();
+
+            this.setScrollAmount(
+                    this.getScrollAmount() + MathUtils.map((float) dragY, 0, height, 0, this.getMaxScroll()));
+        }
+
+        return false;
+    }
+
+    @Override
+    public boolean mouseReleased(double mouseX, double mouseY, int button) {
+        draggingScrollButton = false;
+
+        return super.mouseReleased(mouseX, mouseY, button);
+    }
+
+    private void renderScrollButton(PoseStack poseStack) {
+        float xPos = getScrollButtonXPos();
+        float yPos = getScrollButtonYPos();
+
+        int size = (int) (settingsScreen.width / 65f);
+
+        RenderUtils.drawTexturedRect(
+                poseStack,
+                Texture.SCROLL_BUTTON.resource(),
+                xPos,
+                yPos,
+                0,
+                size,
+                size,
+                0,
+                0,
+                Texture.SCROLL_BUTTON.width(),
+                Texture.SCROLL_BUTTON.height(),
+                Texture.SCROLL_BUTTON.width(),
+                Texture.SCROLL_BUTTON.height());
+    }
+
+    private float getRenderHeight() {
+        return settingsScreen.height - settingsScreen.getBarHeight() * 2;
+    }
+
+    private float getScrollButtonYPos() {
+        float height = getRenderHeight();
+
+        return MathUtils.map(
+                (float) this.getScrollAmount(), 0, this.getMaxScroll(), settingsScreen.getBarHeight() + 4, height - 4);
+    }
+
+    private float getScrollButtonXPos() {
+        return settingsScreen.width / 5.515f;
+    }
+
+    public void reAddEntriesWithSearchFilter(String searchText) {
+        this.clearEntries();
+        this.setScrollAmount(0);
+
+        float renderHeight = 0;
+
+        String lastCategory = "";
+
+        for (Feature feature : FeatureRegistry.getFeatures().stream()
+                .filter(feature -> !(feature instanceof DebugFeature))
```

Comment:
```suggestion
                .filter(feature -> !(feature instanceof DebugFeature) || WynntilsMod.isDevelopmentEnvironment())
```
I think it'd be nice for our devs to easily access debugfeatures in menu.

Reply:
Maybe, but I am not sure if we should do this. Devs can and should be able to use the config command and/or edit the config files, then reload the configs with already existing command.

Reply:
I agree. Or we can hand-edit the files to enable/disable features and recompile.

Reply:
> I agree. Or we can hand-edit the files to enable/disable features and recompile.

Can you clarify which comment you agree with 😅?

Reply:
I agree that it would still be more convenient to use the menu

Reply:
I agree with the agreeableness of all agreements. Agreed? :-)

Seriously, though, I thought that debug features do not need GUI configs. They are a special case tool. If we start adding them in devel builds, it only means the config screen will be not-as-in-production for devel builds, which is bad, for very little gain.